repo2docker/buildpacks/base.py

```diff
@@ -414,12 +415,18 @@ def get_start_script(self):
         """
         return None
 
+    @property
+    def binder_dir(self):
+        if self._binder_dir is None:
+            for dirname in ['binder', '.binder']:
+                if os.path.exists(dirname):
+                    self._binder_dir = dirname
```

Comment:
This will currently assign `.binder` over `binder` if both exists I think. Either add a break statement or reverse the order in the array.

Reply:
yes, I was missing an intended break statement. It looks like we're going for a more explicit rendition in the comments below. Either way, nice catch.